.github/workflows/ci.yml

```diff
@@ -0,0 +1,162 @@
+name: Continuous Integration
+
+on:
+  - push
+  - pull_request
+
+jobs:
+  build:
+    name: ${{ matrix.name }}
+
+    env:
+      MAKEJOBS: "-j3"
+      CHECK_DOC: "0"
+      CCACHE_SIZE: "100M"
+      CCACHE_TEMPDIR: /tmp/.ccache-temp
+      CCACHE_COMPRESS: "1"
+      BASE_OUTDIR: $TRAVIS_BUILD_DIR/out
+      SDK_URL: https://bitcoincore.org/depends-sources/sdks
+      PYTHON_DEBUG: "1"
+      WINEDEBUG: fixme-all
```

Comment:
Please confirm this is correct and note a note to yourself.

Reply:
I'm removing BASE_OUTDIR (travis leftover that survived cleanup) and SDK_URL (will come back with macOS) together with the SDK steps because those all get skipped anyway.

Python and Wine debug statements are good, respectively enabling debugging and suppressing "FIXME" messages.